docs/developing.md

```diff
@@ -158,37 +205,120 @@ Navigate to the `python3` executable, for conda installed via homebrew this may
 
 Save the new interpreter and Apply to the project
 
+##### VS-Code
+
+Open Command Pallete (⇧ ⌘ P), then type `Python: Select interpreter`, then navigate to the `python3` executable for your env.
+
 #### 2. Use a tox env
 
 This is not recommended by the tox maintainers, but you can re-use the existing `tox` environments for other purposes.
 
 First create a tox environment, e.g. by running `tox -e py311`.
 Follow the above steps for selecting an existing environment, and use `${repo_root}/.tox/py311/bin/python3`
 
-#### 3. Let PyCharm manage a fresh one
+#### 3. Let your IDE manage a fresh one
+
+##### PyCharm
 
 PyCharm can also create a new virtual environment for you.
 From the `Add Local Interpreter` menu, select `Environment: new`.
 
 You can choose where the environment is created, by default it will be placed in `${repo_root}/venv`.
 
-### Adding folders to Pycharm's path
+##### VS-Code
+
+VS-Code can also create a new virtual environment for you.
+
+Open Command Pallete (⇧ ⌘ P), then type `Python: Create environment`, then select either `conda` or `venv`. You can also point it to your `requirements.txt` file if you have one to install all dependencies automatically.
+
+### Adding folders to Python path
 
 For some projects you may need to add directories to your PYTHONPATH to make packages importable.
 For example, `caikit/caikit` has a `sample_lib` test fixture which contains a sample implementation of an AI library using caikit.
 The top-level `conftest.py` will take care of adding that fixture to the syspath, but if you want PyCharm to have code-completion and indexing support, you'll need to configure it.
 
+#### PyCharm
+
 To do this in PyCharm, right-click the directory in the project explorer and select `Mark Directory As > Sources Root`
 
 ![](../assets/images/mark_sources.png)
 
-### Running a debugger with PyCharm
+#### VS-Code
+
+Create a `.env` file in the root directory, and add things to that:
+
+```
+PYTHONPATH=tests/fixtures
```

Comment:
does this need to reference the existing path as well? Something like:
```
PYTHONPATH=tests/fixtures;${PYTHONPATH}
```
(I don't have a clue what the answer would be)

Reply:
Hmm, it seems to work as is, it basically adds this path to the `PYTHONPATH` instead of assigning the value to it

Reply:
slick!